gaphor/ui/filemanager.py

```diff
@@ -94,9 +94,12 @@ def load(self, filename):
             self.event_manager.handle(FileLoaded(self, filename))
         except Exception:
             error_handler(
-                message=gettext(
-                    "Error while loading model from file {filename}"
-                ).format(filename=filename),
+                message=gettext("Unable to open model “{filename}”.").format(
```

Comment:
Can we use f-strings with the gettext calls?

Reply:
No. Because f-strings will substitute in place, hence, the gettext text can not be resolved.

E.g. I have a text `"hello {name}"`. When I do `name = "Arjan"; gettext("hello {name}")`, gettext will see the text `"hello Arjan"`, while it should resolve `"hello {name}"` and after that replace `{name}` with `Arjan`.

Reply:
Thanks for the explanation! :smile: